### ISSUE TITLE
Error setting up yayalint

### ISSUE THREAD

**Issue by Zichqec** [Tatakinov/yayalint#14]
I'm trying to configure yayalint in my ghost for the first time, and I got this error.
```
"F:\Stuff\SSP\000_Dev\_yayalint\yayalint.exe" "f:\Stuff\SSP\ghost\Dev\s_the_skeleton\ghost\master\yaya.txt" attempt to index a nil value stack traceback: [C]: in for iterator 'for iterator' [string "yayalint"]:602: in upvalue 'recursive' [string "yayalint"]:655: in upvalue 'recursive' [string "yayalint"]:614: in upvalue 'recursive' [string "yayalint"]:645: in upvalue 'recursive' [string "yayalint"]:970: in upvalue 'interpret' [string "yayalint"]:1154: in local 'main' [string "yayalint"]:1159: in local 'func' [string "yayalint"]:41: in main chunk
```
I've opened [this folder](https://github.com/Zichqec/s-the-skeleton/tree/master/updates/ghosts/s_the_skeleton/ghost/master) of my ghost in vscode, and I'm using `.\yaya.txt` as the path to yaya.txt

**Comment by Tatakinov** [comment]
Maybe fixed in v1.8.2.

**Comment by steve02081504** [comment]
@Tatakinov The latest version of lint seems to have encountered some unimaginable problems when analysing a single line of if.
![图片](https://user-images.githubusercontent.com/31927825/191052353-84071ad4-d2c8-465f-bedd-cefb794b1ecc.png)
![图片](https://user-images.githubusercontent.com/31927825/191052554-d8a55ad0-8b49-4279-992e-1b4e165e52f5.png)
![图片](https://user-images.githubusercontent.com/31927825/191052706-2a846a06-14f3-458a-837a-eb3ffb374e39.png)
![图片](https://user-images.githubusercontent.com/31927825/191052949-2068db09-02f7-40fd-a50e-e23152200ada.png)
![图片](https://user-images.githubusercontent.com/31927825/191052839-e1b5cde1-6fdc-41e8-ab79-513d923b6462.png)

**Comment by Tatakinov** [comment]
maybe fixed in v1.8.3.

**Comment by steve02081504** [comment]
> maybe fixed in v1.8.3.

The fix is working!
Thanks
